.github/workflows/release.yml

```diff
@@ -0,0 +1,117 @@
+name: release
```

Comment:
Binaries will now be built in Google Cloud Build, so maybe we can delete this now?

Reply:
This is fine, though I don't think we want it to be built on each push on `master` it's better to build it on `tag` created.

Reply:
I'll remove the note about "every push..." and we can discuss the best approach for this when I create my deployment doc :)